src/newsroom-context/hooks/useGetTranslationUrl.ts

```diff
@@ -63,8 +64,10 @@ function getTranslationUrl(
         const allowedVisibilityValues = getAllowedTranslationVisibilityValues(currentStory);
 
         const translatedStory = currentStory.translations.find(
-            ({ culture, visibility }) =>
-                culture.locale === localeCode && allowedVisibilityValues.includes(visibility),
+            ({ culture, lifecycle_status, visibility }) =>
+                culture.locale === localeCode &&
+                Story.isPublished(lifecycle_status) &&
+                allowedVisibilityValues.includes(visibility),
```

Comment:
I was wondering if we even need `getAllowedTranslationVisibilityValues` because we don't want to generate alternate links for private or embargo stories anyway? 🤔 As those are supposed to be "not public", right?

Reply:
I can't remember the exact ticket, but we had a discussion, where we agreed on this logic, most likely 'cause it was a requirement for a customer. It was definitely done on purpose, so there should be some reason behind it :)